build-tools/package.json

```diff
@@ -96,7 +96,7 @@
 	},
 	"packageManager": "pnpm@9.15.7+sha512.ed98f9c748442673c46964b70345bd2282c9b305e8eae539b34ab31d6ef24ef8dd59d8b55f27466f705500b009d9c113471cf87e544f3d5036b297330c26e996",
 	"engines": {
-		"node": ">=18.17.1"
+		"node": ">=20.15.1"
```

Comment:
No need to change, but it's annoying that the nvmrc doesn't have the minor, because I had a 20.14.x version available which nvm used, but it wasn't compat with engines.node. Not sure there's a better solution.